micrometer-shim/src/main/java/io/opentelemetry/micrometershim/UnsupportedReadLogger.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.micrometershim;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+final class UnsupportedReadLogger {
+
+  static {
+    Logger logger = Logger.getLogger(OpenTelemetryMeterRegistry.class.getName());
```

Comment:
Just like we don't want people to use our internal classes I don't want to use others' as well :P This logger class seems fine for OTel users

Reply:
Hmm, I had the impression that it could be used inside `MeterRegistry` implementations - but I've double checked it just now, it seems it's only used in `micrometer-core` itself.
Yeah, let's keep using JUL then.